NEWS.md

```diff
@@ -6,6 +6,15 @@ used in the Julia ecosystem. Notable changes will be documented in this file
 for human readability.
 
 
+## Changes when updating to v0.14 from v0.13.x
+
+#### Added
+
+- The `VisualizationCallback` now supports using Makie, so far for 2D TreeMesh only.
+  To use it, a Makie backend has to be loaded and the keyword argument `plot_creator`
+  needs to be set to `Trixi.show_plot_makie`.
```

Comment:
Wouldn't it make sense to export `show_plot_makie` in this case that it needs to be called by a user?

Reply:
Fine with me. Please choose some names that make sense to you (and are sufficiently symmetric for Plots.jl and Makie.jl).